### DISCUSSION TITLE
GitHub Education Application

### DISCUSSION THREAD

**Discussion by theofficialbido** [community/community#65044] [GitHub Education] [answered]
### Hi everyone,

Hello,
I live in Egypt.
My School does not provide E-mails to its students, but my ministry of Education does.
When applying, I get rejected because my photo of the school portal, which contains everything from name to personal id number, does not contain the name of the Ministry of Education.
It does contain the name of my school.
What can I do?

**Comment by arafajs** [comment, marked as answer]
In schools, there is a document called an **attendance document.** This document can be used as proof to activate the service for you.

**Reply by theofficialbido** [reply]
Thanks so much for that,
Hope you have a nice day
